tests/integration/misc/test_misc.py

```diff
@@ -118,111 +118,108 @@ def test_verify_str_arg():
     assert actual == desired
 
 
-def test_get_input_image_tensor():
-    image = torch.tensor(0.0)
-
-    starting_point = image
-    actual = misc.get_input_image(starting_point)
-    desired = image
-    assert actual is not desired
-    ptu.assert_allclose(actual, desired)
-
-
-def test_get_input_image_tensor_content():
-    starting_point = "content"
-    image = torch.tensor(0.0)
-
-    actual = misc.get_input_image(starting_point, content_image=image)
-    desired = image
-    assert actual == ptu.approx(desired)
-
-    with pytest.raises(RuntimeError):
-        misc.get_input_image(starting_point, style_image=image)
+class TestGetInputImageTensor:
```

Comment:
It seems the tests still have a legacy name. Let's rename them 
```suggestion
class TestGetInputImage:
```